src/components/MilestoneConversations.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Avatar from 'react-avatar';
+import moment from 'moment';
+
+import { feathersClient } from './../lib/feathersClient';
+import Loader from './Loader';
+import { getUserName, getUserAvatar } from './../lib/helpers';
+import getNetwork from './../lib/blockchain/getNetwork';
+
+/* eslint no-underscore-dangle: 0 */
+class MilestoneConversations extends Component {
+  static getReadeableMessageContext(context) {
```

Comment:
Would make sense to name this `state` or `milestoneState` to avoid confusion.

Reply:
Also note to self: Replace with constants from the milestone model once rewriting.

Reply:
The reason I've called it context is because the context of a conversation message isn't always the milestone state. For example, there's currently a context called 'replyTo' (not implemented in frontend) and perhaps in the future there will be other contexts.

Agree about replacing with constants